sample/src/main/res/layout/rtl_fragment_view_pager_sample.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Could we rename this file to ```fragment_rtl_view_pager_sample.xml```? A slight nitpick, I just like to have fragment_/activity_/viewholder_/etc as my prefix for xml files.

Reply:
sure thing! 🌮